parse_hyperparameters.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import json
+
+import optuna
+
+
+def value_key(a):
```

Comment:
this function is missing proper variable names, docstring and types. This is why I meant by "typing" is missing